### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by stacietaylorcima**
## Description

This commit adds a PR template to this repo per our discussions in the mentor working group.

This will help make sure our cohort members and mentors can easily get up to speed on the issue being solved in the PR. 

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|   | :bug: Bug fix              |
|   ✓  | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Testing Steps / QA Criteria

- [ ] Provide feedback on what you'd like to see included in a PR

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+|     | Type                       |
+| --- | -------------------------- |
+| ✓   | :bug: Bug fix              |
+|    | :sparkles: New feature     |
+|    | :hammer: Refactoring       |
+|    | :100: Add tests            |
+|    | :link: Update dependencies |
+|    | :scroll: Docs              |
+
```
Comment (laurenmbeatty) [inline pull_request_template.md:27]: I love this table. Totally stealing it.

**Review by stevelikesmusic — APPROVED** [review]
Thanks for adding the PR template @stacietaylorcima!!

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+## Updates
+
+
+### Before
+
+
+### After
+
+
+## Testing Steps / QA Criteria
```
Comment (laurenmbeatty) [inline pull_request_template.md:40]: Maybe Acceptance Criteria (in addition to above)?

**Review by laurenmbeatty — APPROVED** [review]
Thank you! This is a big improvement.

**Pull request merged**